src/scripts/webpack/webpack.build.ts

```diff
@@ -49,7 +49,7 @@ export default (input: WebpackConfigBuilderInput): WebpackConfiguration => {
 
     entry: paths.ENTRY,
 
-    devtool: 'source-map',
+    devtool: config.devTool as any,
```

Comment:
Why is this cast needed?

Reply:
`webpack.Configuration['devtool']` is more complex than `string`

I tried to reproduce it with `io-ts` but it kept complaining => I gave up